WebHostLib/static/assets/player-settings.js

```diff
@@ -101,11 +101,16 @@ const buildOptionsTable = (settings, romOpts = false) => {
 
     // td Left
     const tdl = document.createElement('td');
+    tdl.setAttribute('data-tooltip', settings[setting].description);
     const label = document.createElement('label');
-    label.setAttribute('for', setting);
-    label.setAttribute('data-tooltip', settings[setting].description);
     label.innerText = `${settings[setting].displayName}:`;
+    label.setAttribute('for', setting);
+    const question_span = document.createElement('span');
+    question_span.setAttribute('class', 'interactive')
+    question_span.innerText = ` (?)`;
+
     tdl.appendChild(label);
+    label.appendChild(question_span);
```

Comment:
i think you want regular single quotes there.

Also here you have
`<label>...<span> (?)</span></label>` while in generate you have
`<label>...</label> <span>(?)</span>`